### DISCUSSION TITLE
[Feature] More mobile friendly web UX

### DISCUSSION THREAD

**Discussion by MozPri** [immich-app/immich#4798] [Feature Request]
### The feature

Shared albums are just big single-column photos with too big album title on a mobile web browser.
It would be nice if the web UX design was similar to that of the mobile app. 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by aisbergde** [comment]
And I would say, the Web UI is better than the mobile UI and I would like the mobile view more similar to the web view. Especially in the Web the thumbnails are showing the whole picture in original hight wight ratio. But the mobile client (Android) is cropping images. I don't like this.

**Reply by MozPri** [reply]
You can enable dynamic layout in the app (settings/photo grid) to show photos without cropping.

**Reply by aisbergde** [reply]
That was, what I wanted. I didn't know, what dynamic layout means and did not enable before. Now I know.

And, maybe the current feature request is more about using the web UI on a tablet or on a touch screen device.

**Comment by histefanhere** [comment]
The one-column view on mobile web is still there in 2025 and it's horrendous 💀 It's especially a bad look when I use immich to share an album of photos with friends after an event and send the group a share link but the view is just a looooong single file list of photos. There are wide margins on either side that aren't necessary on mobile and the photos could be arranged two or even 3 side-by-side (i.e. made smaller)

**Comment by jjspierx** [comment]
Viewing higher res versions of the file is also a 2-press process.  When scrolling through the album a press on the screen "selects" the image, but doesn't do anything, until you press again to open the image.  The default "press" action I would think should open the image in the asset view and not just select the asset.

**Comment by wtangofoxtrot** [comment]
Leaving some images here to show how bad the web view is when compared to alternatives like nextcloud:

| Immich iOs App  | Immich Web | Nextcloud Web |
| ------------- | ------------- | ------------- |
| ![image](https://github.com/user-attachments/assets/29c6f4e0-4083-4121-8abb-dcbb204f50d3) | ![image](https://github.com/user-attachments/assets/959a1a6f-fe8f-4ae0-ad83-c4c6e16a2f7a)  | ![image](https://github.com/user-attachments/assets/b2b25afb-63bd-4339-895b-44df6988ee0c) |

**Reply by jzavalaneri** [reply]
Hoping this gets fixed. The Immich web on mobile is not very good and it's what stops me from sharing albums with friends and family.  😆

**Comment by YarosMallorca** [comment]
Will be fixed by #16902
